Traceback (most recent call last):
  File "/Users/runner/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/site-packages/jupyter_cache/executors/basic.py", line 163, in execute
    executenb(nb_bundle.nb, cwd=tmpdirname, timeout=timeout)
  File "/Users/runner/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/site-packages/nbconvert/preprocessors/execute.py", line 737, in executenb
    return ep.preprocess(nb, resources, km=km)[0]
  File "/Users/runner/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/site-packages/nbconvert/preprocessors/execute.py", line 405, in preprocess
    nb, resources = super(ExecutePreprocessor, self).preprocess(nb, resources)
  File "/Users/runner/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/site-packages/nbconvert/preprocessors/base.py", line 69, in preprocess
    nb.cells[index], resources = self.preprocess_cell(cell, resources, index)
  File "/Users/runner/hostedtoolcache/Python/3.7.8/x64/lib/python3.7/site-packages/nbconvert/preprocessors/execute.py", line 448, in preprocess_cell
    raise CellExecutionError.from_cell_and_msg(cell, out)
nbconvert.preprocessors.execute.CellExecutionError: An error occurred while executing the following cell:
------------------
from scipy.stats import norm
from scipy.integrate import quad

Ï• = norm()
value, error = quad(Ï•.pdf, -2, 2)  # Integrate using Gaussian quadrature
value
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
[0;32m<ipython-input-3-0c927c4af260>[0m in [0;36m<module>[0;34m[0m
[0;32m----> 1[0;31m [0;32mfrom[0m [0mscipy[0m[0;34m.[0m[0mstats[0m [0;32mimport[0m [0mnorm[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      2[0m [0;32mfrom[0m [0mscipy[0m[0;34m.[0m[0mintegrate[0m [0;32mimport[0m [0mquad[0m[0;34m[0m[0;34m[0m[0m
[1;32m      3[0m [0;34m[0m[0m
[1;32m      4[0m [0mÏ•[0m [0;34m=[0m [0mnorm[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      5[0m [0mvalue[0m[0;34m,[0m [0merror[0m [0;34m=[0m [0mquad[0m[0;34m([0m[0mÏ•[0m[0;34m.[0m[0mpdf[0m[0;34m,[0m [0;34m-[0m[0;36m2[0m[0;34m,[0m [0;36m2[0m[0;34m)[0m  [0;31m# Integrate using Gaussian quadrature[0m[0;34m[0m[0;34m[0m[0m

[0;31mModuleNotFoundError[0m: No module named 'scipy'
ModuleNotFoundError: No module named 'scipy'

